components/vertical-navigation/private/item.jsx

```diff
@@ -34,6 +34,7 @@ const Item = (props) => (
 			href={props.item.url || 'javascript:void(0);'} // eslint-disable-line no-script-url
 			className="slds-nav-vertical__action"
 			aria-describedby={props.categoryId}
+			aria-current={props.isSelected ? true : null}
```

Comment:
Could this be `undefined` instead of `null`?

Reply:
I initially used `undefined` but then saw https://github.com/salesforce/design-system-react/blob/master/components/progress-indicator/private/step.jsx#L169.  So then I changed it to `null` to match.  I can change it to `undefined` if you have a strong preference.